### PR TITLE
make CustomHighlightPlugin usable outside of plugin

### PR DESCRIPTION
**Pull request by gavlig**
Tried to set custom materials in my project and failed because it has a private field. Suggesting this change or requesting wisdom to overcome this build error:
Screenshot from 2022-06-08 15-21-05

### CODE REVIEW

**Review by aevyrie — APPROVED** [review]

**Comment by aevyrie** [comment]
Thanks for the contribution. Could you please respond to this before I merge?
https://github.com/aevyrie/bevy_mod_picking/issues/138

I'm in the process of relicensing, but I will need explicit approval until the license has been updated.

**Comment by gavlig** [comment]
done!

**Pull request merged**
